root/static/scripts/release-editor/fields.js

```diff
@@ -394,6 +394,10 @@ class Medium {
             return !self.canHaveDiscID() && (self.hasExistingTocs() || hasPregap() || hasDataTracks());
         });
 
+        this.hasTooEarlyFormat = ko.computed(function () {
+            return !!self.formatID() && !!self.release && !!self.release.earliestYear() && (self.release.earliestYear() < MB.formatDates[self.formatID()]);
```

Comment:
To fix `self.release` being undefined, you may just need to pass `release` to the `fields.Medium` constructor in whichever test was failing. Here it might be better to remove the check and have it fail immediately, since the computed observable won't work correctly if it only has a subscription on `formatID`.